pkg/observer/controller.go

```diff
@@ -296,11 +297,14 @@ func (c *controller) safeToStartCycle() bool {
 		return false
 	}
 
+	// cluster_autoscaler_last_activity values will update every ~30 seconds in non-scaling scenario
```

Comment:
Where does the 30 seconds come from? Is this the same as the prometheus scrape_interval?

Reply:
Yes this is from the `scrape_interval` attribute

Reply:
Yea `scrape_interval` could change depending on how a user has configured prometheus to scrape cluster-autoscaler - but we could have a CLI flag to change this, with a sane default of 30 or 40 seconds.